docs/examples/LOCAL_LLM_GUIDE.md

```diff
@@ -0,0 +1,111 @@
+# Local LLM Guide with Ollama server
+
+## 0. Install Ollama:
+run the following command in a conda env with CUDA etc.
+
+Linux:
+```
+curl -fsSL https://ollama.com/install.sh | sh
+```
+Windows or macOS: 
+
+- Download from [here](https://ollama.com/download/)
+
+## 1. Install Models:
+Ollama model names can be found [here](https://ollama.com/library) (See example below)
+
+![alt text](images/ollama.png)
```

Comment:
looks like we lost `example.png`. Should we delete it?

Reply:
Yes, it is no longer necessary.